src/store/importexport/electronExport.tsx

```diff
@@ -77,7 +77,7 @@ export async function electronExport(
   ): FileResponse => {
     return {
       message: fileName,
-      fileURL: 'file:////' + fullpath,
+      fileURL: 'file:///' + fullpath,
```

Comment:
This seemed to be a separate issue that we found this morning where it wouldn't save the download file. (So there may be four issues in the commit. I wonder if it would be possible to make four separate smaller commits where the changes in each commit are more tightly related to each other and to one Jira issue.

Reply:
created this issue so this can be tested [TT-4982](https://jira.sil.org/browse/TT-4982).